make.js

```diff
@@ -185,8 +185,6 @@ CLI.build = function(/** @type {{ task: string }} */ argv)
     if (process.env.TF_BUILD) {
         fail('Please use serverBuild for CI builds for proper validation');
     }
-
-    callGenTaskDuringBuild = true;
```

Comment:
please restore the generated tasks in _generated\* for the maven PR, we'll check in those changes seperately

Reply:
Updated!